corehq/apps/app_manager/templates/app_manager/partials/releases/releases_deploy_modal.html

```diff
@@ -151,10 +151,9 @@ <h4 class="panel-title">
                             </li>
                             <li>
                               {% blocktrans %}
-                                From CommCare ODK's install screen menu, bring up the
-                                application menu by hitting the three dots in the upper
-                                right or by hitting the menu button on your phone to
-                                select Offline Install
+                                Open CommCare Homepage, click on three dots on
+                                top right and go to "Update App", then select
+                                "Offline Update" from the menu (three dots)
```

Comment:
I know you're using the wording that was provided, but I think this should either read
"Open the CommCare homepage," or "Open CommCare,"

My preference would also be to remove "(three dots)" at the end; I find it more confusing than helpful.

Reply:
Thanks! I shared your feedback with support, and they updated the wording. What do you think of this? 950ff7812924eda47d49d4968de0b8e41b452e1c